src/Calculator/Resources/en-US/Resources.resw

```diff
@@ -2729,6 +2729,10 @@
     <value>Microsoft Software License Terms</value>
     <comment>Displayed on a link to the Microsoft Software License Terms on the About panel</comment>
   </data>
+  <data name="PreviewTag.Text" xml:space="preserve">
+    <value>Preview</value>
+    <comment>Preview tag for upcoming features</comment>
```

Comment:
If someone has an alternative wording that they'd prefer. Feel free to comment and I can update.

Reply:
I think that you've written is OK. I might write "Label displayed next to upcoming features"

The important thing about the comment is to help a human translator understand how the string will be used, since they probably won't see the string in context. In this case we'll want them to understand that we mean "preview" in the sense of "beta feature" rather than in the some other sense (like "print preview"). It's also helpful to describe whether the text is for display, a screen reader, or something else.

Reply:
Good suggestion, updated.